Makefile.sdk.mk

```diff
@@ -577,6 +580,7 @@ build/fs-env-%.rc:
 		FRIDA_ASAN=$(FRIDA_ASAN) \
 		FRIDA_ENV_NAME=fs \
 		FRIDA_ENV_SDK=none \
+		PYTHON3=$(PYTHON3) \
```

Comment:
This also needs to be done in the other places where we invoke the script – using `git grep setup-env.sh` is probably the easiest to find them.